creator/studies/migrations/0029_add_populate_domain.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 2.2.24 on 2021-11-02 18:23
+
+"""
+This migration creates the domain attribute for the Study model and also
+populates it using the spreadsheet found in this migrations folder.
+"""
+
+from django.db import migrations, models
+
+import pandas as pd
+import os
+
+
+def set_domain(apps, schema_editor):
+    """
+    Populate domain for all Study objects using the provided data.
+    """
+    df = pd.read_csv(
+        os.path.abspath("creator/studies/migrations/study_metadata.csv")
+    )
+    domains = pd.Series(df.domain.values, index=df.kf_id).to_dict()
+    Study = apps.get_model("studies", "Study") 
+    mappings = {
+        "Birth Defect": "BIRTHDEFECT",
+        "Other": "OTHER",
+        "Cancer": "CANCER",
+        "Cancer,Birth Defect": "CANCERBIRTHDEFECT",
+        "COVID-19": "COVID19",
+    }
+
+    for study in Study.objects.all():
+        kf_id = study.kf_id
+        if kf_id in domains:
```

Comment:
If a study isn't found in the migration file or doesn't have a domain set, let's set fallback to setting domain to "Unknown"